### PR TITLE
Fix the issues pointed out by staticcheck

### PR DESCRIPTION
**Pull request by vagababov**
executed `staticcheck ./...` and fixed issues pointed out there.

/assign @vaikas-google

### CODE REVIEW

File: configmap/manual_watcher.go [hunk 1]
```diff
@@ -40,10 +40,10 @@ func (w *ManualWatcher) Watch(name string, o Observer) {
 	defer w.m.Unlock()
 
 	if w.observers == nil {
-		w.observers = make(map[string][]Observer)
+		w.observers = make(map[string][]Observer, 1)
 	}
 
-	wl, _ := w.observers[name]
+	wl := w.observers[name]
```
Comment (vaikas-google) [inline configmap/manual_watcher.go]: can we just delete this and do:
w.observers[name] = append(w.observers[name], o)
?

**Comment by vaikas-google** [comment]
/hold
/lgtm
/approve

Just one question, remove the hold if not useful.

File: configmap/manual_watcher.go (hunk 1 above)
Comment (vagababov) [reply configmap/manual_watcher.go]: Done.

**Pull request merged**